src/ui/molecules/stepper/stepper.scss

```diff
@@ -66,24 +66,38 @@ $colors: (
     }
   }
 
-  .okp4-stepper-content {
-    grid-area: content;
-    overflow-y: auto;
+  .okp4-stepper-container {
     width: 100%;
-    padding: 2px;
+    height: 100%;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+
+    @include with-theme() {
+      background-color: themed(primary-background);
```

Comment:
Just a question on the background: wasn't it supposed to be the same color as the cards in the faucet?

Reply:
And I have this problem on the `Label` story: 
![Capture d’écran 2022-08-23 à 08 16 31](https://user-images.githubusercontent.com/35332974/186084471-e8760e0f-3e45-416f-82cb-a774198491b3.png)